README.md

```diff
@@ -41,7 +42,7 @@ available at [http://localhost:3000/](http://localhost:3000/).
 
 #### Unit Test
 
-The unit tests are powered by [chai](http://chaijs.com/) and [mocha](http://mochajs.org/).
+The unit tests are powered by [Jest](https://facebook.github.io/jest/), [Enzyme](http://airbnb.io/enzyme/) and [Sinon.JS](http://sinonjs.org/)
```

Comment:
The description in Readme.md was wrong, the default unit test runner was jest, not mocha. There's one line - "test": "jest" in package.json. So that the unit test did not use mocha.